go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/gogoproto v1.7.0
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v10 v10.0.0-beta.2
+	github.com/cosmos/ibc-go/modules/apps/callbacks/v10 v10.0.0-rc.2
```

Comment:
It's unclear to me if we're fine just using this directly through our ibc-go dependency or if we need to depend on it separately like this.

Reply:
Seems like callbacks aren't included in ibc-go/v10 and have to be imported separately. Assuming that this is because callbacks are optional.

Fwiw the same thing happens with the wasm light client. It has separate releases and thus separate imports